spec/request/authentication_spec.rb

```diff
@@ -5,50 +5,14 @@
 RSpec.describe 'Sign in users', type: :request do
   let(:password) { SecureRandom.hex(16) }
 
-  context 'An Admin' do
-    let(:admin) { FactoryBot.create(:admin_user, password: password) }
-
-    it 'can sign in with valid credentials' do
-      post new_admin_user_session_path, params: {
-        'admin_user[email]' => admin.email,
-        'admin_user[password]' => password
-      }
-
-      expect(response.status).to eq(302)
-      expect(response.location).to eq(authenticated_admin_user_root_url)
-    end
-
-    context 'without a valid CSRF token' do
-      # We're specifically interested in checking the forgery protection in these
-      # tests. It's not usually enabled in the test environment so we'll turn it on
-      # here.
-      before do
-        ActionController::Base.allow_forgery_protection = true
-      end
-
-      after do
-        ActionController::Base.allow_forgery_protection = false
-      end
-
-      it 'is rejected' do
-        expect do
-          post new_admin_user_session_path, params: {
-            'admin_user[email]' => admin.email,
-            'admin_user[password]' => password
-          }
-        end.to raise_error(ActionController::InvalidAuthenticityToken)
-      end
-    end
-  end
-
   context 'A User' do
     let(:user) { FactoryBot.create(:user, password: password) }
 
     it 'can sign in with valid credentials' do
-      post new_admin_user_session_path, params: { 'user[email]' => user.email, 'user[password]' => password }
+      post new_user_session_path, params: { 'user[email]' => user.email, 'user[password]' => password }
 
       expect(response.status).to eq(302)
-      expect(response.location).to eq(authenticated_user_root_url)
+      expect(response.location).to eq('http://www.example.com/')
```

Comment:
Why 'http://www.example.com'??

Reply:
that's what the test.rb is configured to.

Reply:
changed to root_url